app/src/main/java/com/example/weatherapp/data/Constant.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.weatherapp.data
+
+const val BASE_URL ="https://weather.exam.bottlerocketservices.com/"
+const val GET_CITY ="Cities"
```

Comment:
possible type with Uppercase 
Cities
??

Reply:
got it, will fix